shared/loadgenerator/loadgenerator.go

```diff
@@ -107,6 +116,17 @@ func (g *GeneratorOptions) CreateRunnerOptions(resolvableDomain bool) *fhttp.HTT
 		HTTPOptions:        *o,
 		AllowInitialErrors: g.AllowInitialErrors,
 	}
+
+	if len(g.FileNamePrefix) != 0 {
+		dir := prow.GetLocalArtifactsDir()
+		if err := common.CreateDir(dir); err != nil {
+			return nil
```

Comment:
shouldn't let the function return an error as well in this case?

Reply:
sg. Updated